geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionClearDUnitTest.java

```diff
@@ -104,14 +102,21 @@ private void initClientCache() {
     region.registerInterestForAllKeys(InterestResultPolicy.KEYS);
   }
 
-  private void initDataStore() {
-    getCache().createRegionFactory(getRegionShortCut())
-        .setPartitionAttributes(new PartitionAttributesFactory().setTotalNumBuckets(10).create())
-        .addCacheListener(new CountingCacheListener())
-        .create(REGION_NAME);
+  private void initDataStore(boolean withListener, boolean withWriter) {
```

Comment:
There is "PartitionedRegionLoaderWriterDUnitTest" specific to testing loader and writers on PR. Probably we should move these tests over there.

Reply:
I read the PartitionedRegionLoaderWriterDUnitTest. That is only to verify compatibility when creating region. We are testing the clear() operation's behavior.

Reply:
You can add/extend the test as per your needs. The tests seems to be added for CacheLaderWriter and may be during that time, the need is to test create region.